lib/utils.js

```diff
@@ -56,9 +43,9 @@ utils.isPath = function isPath(pattern, opts) {
  * @return {Function}
  */
 
-utils.hasPath = function hasPath(pattern, opts) {
+utils.pathContains = function(pattern) {
   return function(fp) {
-    return utils.unixify(pattern, opts).indexOf(fp) !== -1;
```

Comment:
> Is there coverage somewhere in the tests for ensuring it happens as needed?

let's hold off. I'd rather you be comfortable with the changes than just merge them in. all of it is minor anyway